lessons/prog_langs/python_3_11_4/philosophy.yaml

```
@@ -0,0 +1,59 @@
+node_ref: "prog_langs.python_3_11_4.philosophy"
+content:
+  - type: "paragraph"
+    value: |
+      # Mindset and philosophy of devs
+
+      Education and development can seem challenging if you go alone and without a plan and guiding lights. In this lesson, we discuss how important your mental health is during education, what Python's philosophy is, and how it will help us.
+
+      ## Mental toughness
+      There are countless numbers of ways people can learn. Pacing yourself through the course as if preparing for a full marathon is one of the least stressful and harmful strategies. Slowly building a habit of learning every day and incorporating the study process into your routine leads to several benefits:
+      You have time to digest the information and rest well between learning sessions. It's very easy to get stressed out when you spend countless hours fighting with something to find out the problem was your reduced attentiveness. The next day your brain, after a good night of sleep, will solve any problem. But you need to give it some rest before!
```

опять ничего не понятно, очень замудрено 
```suggestion
      Taking the time to process information and ensuring sufficient rest between learning sessions is essential. It's easy to become stressed when spending endless hours struggling with a task, only to realize that the issue stems from decreased attentiveness. By allowing your brain to rest and rejuvenate through a good night's sleep, you enable it to effectively solve problems the next day. Therefore, it is important to prioritize rest and provide your brain with the necessary downtime before tackling challenges.
```